common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2886,6 +2886,12 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_UTF8_STRING" : {
+    "message" : [
+      "Invalid UTF8 byte sequence found in string: <str>."
+    ],
+    "sqlState" : "HV00A"
```

Comment:
@srielau HV00A is `invalid string format`
should we be going with this or something else?

Reply:
update: consulted with @srielau and decided to use: 22029, which is `noncharacter in UCS string`